tools/state-viewer/src/cli.rs

```diff
@@ -668,18 +667,17 @@ impl clap::ValueEnum for ViewTrieFormat {
 
 /// Possible record types in a state trie.
 #[derive(Clone)]
-#[repr(u8)]
 pub enum RecordType {
-    Account = col::ACCOUNT,
-    ContractCode = col::CONTRACT_CODE,
-    AccessKey = col::ACCESS_KEY,
-    ReceivedData = col::RECEIVED_DATA,
-    PostponedReceiptId = col::POSTPONED_RECEIPT_ID,
-    PendingDataCount = col::PENDING_DATA_COUNT,
-    PostponedReceipt = col::POSTPONED_RECEIPT,
-    DelayedReceiptIndices = col::DELAYED_RECEIPT_INDICES,
-    DelayedReceipt = col::DELAYED_RECEIPT,
-    ContractData = col::CONTRACT_DATA,
+    Account,
+    ContractCode,
+    AccessKey,
+    ReceivedData,
+    PostponedReceiptId,
+    PendingDataCount,
+    PostponedReceipt,
+    DelayedReceiptIndices,
+    DelayedReceipt,
+    ContractData,
```

Comment:
Not sure what happened exactly but I love it!

Reply:
oopsie, that was actually a mistake. I've removed it and thought it will cause a compile-time error if that is wrong, but it turns out casting to `u8` still worked. Changes this to have a single record for DelayedReceiptOrIndices.